production/user_guide.md

```diff
@@ -175,8 +176,53 @@ As shown above, in the field labeled "Resource types requested," type "Coverage.
 
 If you’d like to use the command line or implement this API call in code, look in the `Curl` section (shown in the image above) for the request you just made. Not far below that, you can see the response: an `HTTP 202 Accepted` giving a link in the content-location header for status information on your Coverage job.
 
-### 5. Getting your data
-There are two steps to retrieving the requested Coverage data:
+### 5. Using the `_since` parameter
+The `_since` parameter lets you filter your requested bulk data requests by the date when it was updated. This means that instead of receiving all your historical data each time you request data from the API, you will instead be able to enter the date since you last requested data. The API will return data updated between your `_since` input date and the present.
+
+Before using `_since` for the first time, pull your historical data. Using the `_since` parameter subsequently comprises two steps:
+
+1. Input a date in the correct format.
+2. Start the job to acquire data from an endpoint.
+
+#### a. Pull your historical data.
+Before using `_since` for the first time, we recommend that you retrieve all historical data from the BCDA bulk data endpoints (do not use `_since`). See [Making Your First Requests for Data](https://bcda.cms.gov/production/user-guide/#making-your-first-requests-for-data) for step-by-step instructions on how to pull your historical data.
+
+Once you have retrieved your historical data, we recommend using the date of your most recent pull for `_since`. You may retrieve this date by viewing the [_transactionTime_](https://hl7.org/Fhir/uv/bulkdata/export/index.html#response---complete-status) from your last `_since` call.
+
+By retrieving your historical data before calling `_since`, this ensures that there will be no gaps in the claims data delivered from BCDA.
+
+**Note: Do not input dates before 02-12-2020 into `_since`. Limitations of the Beneficiary FHIR Data (BFD) Server prevent data before 02-12-2020 from being tagged correctly.  For more details, see the [Advanced User Guide](https://bcda.cms.gov/production/technical-user-guide/#filtering-your-data-with-_since).**
```

Comment:
Does this `note` contradict this: https://github.com/CMSgov/bcda-static-site/pull/42/files#diff-7b5449ca528afb4b448e5e149523f3dcR682

Reply:
We chose to keep the non-technical version vague without use cases to encourage them to not use it before the date. We included more specific use cases in the advanced technical version. I think the use cases qualify as 'not being tagged correctly'

Reply:
I'm worried that (1) they see contradicting usage advice on two different pages, and (2) we expose limitations in BFD by saying `not being tagged correctly`.